### PR TITLE
[1LP][RFR][NOTEST] Move manual login cases to test_auth_manual.py

### PR DESCRIPTION
**Pull request by john-dupuy**

### CODE REVIEW

File: cfme/tests/test_login.py [hunk 1]
```diff
@@ -277,3 +126,55 @@ def test_multiregion_displayed_on_login(context, setup_multi_region_cluster, mul
         login_view = navigate_to(rapp.server, 'LoginScreen')
         assert login_view.is_displayed
         assert 'Remote' in login_view.details.region.text
+
+
+@pytest.mark.parametrize('context', [ViaUI])
+@pytest.mark.meta(blockers=[BZ(1632718)])
+@test_requirements.auth
+def test_update_password(context, request, appliance):
```
Comment (mshriver) [inline cfme/tests/test_login.py]: Please mark this for browser isolation as well, because of its login/logout pattern.

Comment (john-dupuy) [reply cfme/tests/test_login.py]: done

File: cfme/tests/test_login.py [hunk 2]
```diff
@@ -277,3 +126,56 @@ def test_multiregion_displayed_on_login(context, setup_multi_region_cluster, mul
         login_view = navigate_to(rapp.server, 'LoginScreen')
         assert login_view.is_displayed
         assert 'Remote' in login_view.details.region.text
+
+
+@pytest.mark.parametrize('context', [ViaUI])
+@pytest.mark.meta(blockers=[BZ(1632718)])
```
Comment (sshveta) [inline cfme/tests/test_login.py]: This BZ is verified . It can be removed .
This can be added instead @pytest.mark.meta(automates=[BZ(1632718)])

Comment (john-dupuy) [reply cfme/tests/test_login.py]: done

**Comment by sshveta** [comment]
PRT failed 
`  with pytest.raises(Exception, match=error_message):
>           appliance.server.login(user)
E           Failed: DID NOT RAISE <class 'Exception'>`

**Comment by john-dupuy** [comment]
@sshveta I am just moving test cases around in this PR. I will update to fix failures in a separate PR

**Pull request merged**